openapi.yaml

```diff
@@ -729,7 +728,7 @@ paths:
                             name:
                               type: string
                               description: >-
-                                Name of the plan. Is allowed to be *case
+                                Name of the plan. It MUST be accepted *case
```

Comment:
I think this is not completely correct grammatically. The "case sensitive" must become an adverb, but "must be accepted case insensitively" sounds also bit weird. Do we have native English speakers that can we consult on this?

Reply:
Hmm, not sure. Maybe:

> Format names MUST be accepted in a case-insensitive manner throughout the API.

Reply:
yeah it's probably safer to workaround it that way :smile:

Reply:
Changed.